.github/workflows/_reusable_app_release.yml

```diff
@@ -7,6 +7,9 @@ on:
       is_cloud_build:
         type: boolean
         default: true
+      countly_enabled:
+        type: boolean
+        default: true
```

Comment:
you're missing c1_c2_c3_prod workflow. either we don't provide default here and we specify on every workflows that call it or we provide default but we skip the workflows that use the default

Reply:
fixed: ea9d155222c58c3f76230669051b3c17a3713aa9